OrderItem.java

```diff
@@ -0,0 +1,54 @@
+//
+// Team 1: Spilled Coffee
+// Create order item objects from order CSV fields (no time)
+//
+
+public class OrderItem {
+    // object properties
+    private String date;
+    private String cust_email;
+    private String cust_location;
+    private String product_id;
+    private int quantity;   
+
+    // OrderItem constructor
+    public OrderItem (String date, String cust_email, String cust_location, String product_id, int quantity) {
+        this.date = date;
+        this.cust_email = cust_email;
+        this.cust_location = cust_location;

```

Comment:
We might want to make sure that the zip has a qualifier as well, 00000 is a nonsense one that can be taken in and will mess with the save/load because it will save it as just 0